services/app-api/src/postgres/contractAndRates/parseContractWithHistory.ts

```diff
@@ -280,7 +254,8 @@ function contractWithHistoryToDomainModel(
             contractRevisions[contractRevIndex + 1]
 
         // Reverse rateRevisions so it is in DESC order.
-        const rateRevisions = contractRev.rateRevisions.reverse()
+        const rateRevisions =
+            contractRev.relatedSubmisions[0].submissionPackages
```

Comment:
@macrael did this end up being right? This was my code I think and I remember feeling confused if this was the right way to get the rate revisions on a contract rev

Reply:
I think that's still right, that's more code that will disappear once we drop all the contractRevisions.rateRevisions code which is a follow on PR to this work

Reply:
ticket?